tests/fixtures/payload.py

```diff
@@ -152,12 +154,16 @@
 end_datetime = datetime(2001, 1, 31)
 
 
-def random_datetime() -> datetime:
-    return start_datetime + timedelta(
+def random_datetime_str() -> str:
+    random_datetime = start_datetime + timedelta(
         seconds=random.randint(0, int((end_datetime - start_datetime).total_seconds())),
         microseconds=random.randint(0, 999999),
     )
 
+    format = random.choice(datetime_utils.available_formats)
+
+    return random_datetime.strftime(format)
```

Comment:
why can't I leave a suggestion :(

Reply:
```suggestion
    fmt = random.choice(datetime_utils.available_formats)
    if 'z' in format:
        random_datetime = random_datetime.replace(tzinfo=timezone(offset=timedelta(hours=random.randint(-12, 12))))
    dt_str = random_datetime.strftime(fmt)
    return dt_str
```

also we need to `from datetime import timezone`

Reply:
nice, you just triggered a local implementation bug. I'll fix it

Reply:
Well, it wasn't really a bug. But in core we didn't allow timezones without colons. E.g. `+09:00` was accepted but `+0900` was not. 
I have added this flexibility to add both now in qdrant/qdrant#3729 and qdrant/qdrant#3730, and it works well now